### PR TITLE
Processing traces removal

### LINKED ISSUE #10697: A lot of traces during bootstrap process
When I bootstrap compiler I get a lot of traces which for me non informative and overlap important information. My suggestion to just remove this flood or print these traces only in verbose mode.
```
Processing C:/qx/qooxdoo/source/class/qx/ui/window/Desktop.js ...
Processing C:/qx/qooxdoo/source/class/qx/ui/window/IDesktop.js ...
Processing C:/qx/qooxdoo/source/class/qx/ui/window/IWindowManager.js ...
Processing C:/qx/qooxdoo/source/class/qx/ui/window/Manager.js ...
Processing C:/qx/qooxdoo/source/class/qx/ui/window/MDesktop.js ...
Processing C:/qx/qooxdoo/source/class/qx/ui/window/Window.js ...
Processing C:/qx/qooxdoo/source/class/qx/ui/window/__init__.js ...
Processing C:/qx/qooxdoo/source/class/qx/ui/__init__.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/AliasManager.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/Animation.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/Base64.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/ColorUtil.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/DeferredCall.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/DeferredCallManager.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/Delegate.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/DisposeUtil.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/DynamicScriptLoader.js ...
Processing C:/qx/qooxdoo/source/class/qx/util/EditDistance.js ...
```

OS: Windows 10, Ubuntu 22.04.
qooxdoo version: 7.7.1

To reproduce:

1. Download qooxdoo
2. npm ci
3. ./bootstrap-compiler

### PR DESCRIPTION
**Pull request by goldim**
Temporary resolved #10697 . Any other suggestions are welcomed. Please notice `verbose` flag is true during bootstrap process and it is out of box. Sorry, these traces are annoying and non informative.

### CODE REVIEW

**Review by hkollmann — APPROVED** [review]

File: source/class/qx/tool/cli/commands/Compile.js [hunk 1]
```diff
@@ -789,9 +789,9 @@ Framework: v${await this.getQxVersion()} in ${await this.getQxPath()}`);
       }
 
       for (let filename of classFiles) {
-        if (this.argv.verbose) {
-          qx.tool.compiler.Console.info(`Processing ${filename} ...`);
-        }
+        // if (this.argv.verbose) {
```
Comment (johnspackman) [inline source/class/qx/tool/cli/commands/Compile.js:792]: How about you just delete the lines altogether, instead of commenting them out?

**Review by johnspackman — CHANGES_REQUESTED** [review]

File: source/class/qx/tool/cli/commands/Compile.js (hunk 1 above)
Comment (goldim) [reply source/class/qx/tool/cli/commands/Compile.js:792]: Whatever, I just saw such kind of these traces commented. I thought someone from us will need this for some debug actions.

**Review by cboulanger — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Comment by goldim** [comment]
@johnspackman hello, could you check this PR please and there is some new info in gitter chat too.

**Review by johnspackman — APPROVED** [review]

**Comment by johnspackman** [comment]
@goldim sorry Dmitry, got run out at work ... 🙄

**Pull request merged**